style/main.scss

```diff
@@ -1,81 +1,61 @@
 $o-footer-is-silent: false;
 $o-fonts-is-silent: false;
-$o-typography-is-silent: false;
-@import '../bower_components/o-colors/main';
-@import '../bower_components/o-fonts/main';
-@import '../bower_components/o-footer/main';
-@import '../bower_components/o-typography/main';
-@include oColorsSetColor('claret-inverse', #4f1828);
-$sans-serif: oFontsGetFontFamilyWithFallbacks(MetricWeb);
 
-html {
-  @include oColorsFor(page, background);
-  height: 100%;
-}
+@import 'o-colors/main';
+@import 'o-fonts/main';
+@import 'o-footer/main';
+@include oColorsSetColor('claret-inverse', #4f1828);
 
-body {
-  // Remove space around the document
-  @include oColorsFor(page, background);
-  margin: 0;
-  height: 100%;
-}
+$sans-serif: oFontsGetFontFamilyWithFallbacks(MetricWeb);
 
-// scss-lint:disable IdSelector
-#kat-root {
-  height: 100%;
+// Add this class to your main page wrapper
+.kat-root {
+  min-height: 100%;
+  position: relative;
 }
 
+// The footer styling
 .kat-footer {
+  bottom: 0;
   font-family: $sans-serif;
+  left: 0;
   position: absolute;
   width: 100%;
-  bottom: 0;
-  left: 0;
-  border: 0;
 
-  &__wrapper {
-    min-height: 100%;
-    position: relative;
-  }
-
-  &__contact {
-    &-wrapper {
-      background-color: oColorsGetPaletteColor("claret");
-      padding: 15px 0;
+  .o-footer__legal-links li {
```

Comment:
Should [each link](https://github.com/Financial-Times/kat-footer/pull/25/files#diff-ac0bbfa885fc409b83e4dcbfda648441R13) have an `footer__legal-link` class instead of styling on the `li` element?